src/payment/instrument/supported-payment-instruments.ts

```diff
@@ -5,49 +5,49 @@ interface SupportedInstruments {
 }
 
 const supportedInstruments: SupportedInstruments = {
-    scheme_adyenv2: {
+    'adyenv2.scheme': {
```

Comment:
what's the reason for this change? having dots in the key might make it harder to construct the object.

Reply:
I did this suggestion following a @icatalina DM (from slack) and he tell me that it's ok to keep as this. If I compose like: 
```
adyenv2: {
   scheme: { }
}
```

Then I will need to modify the type of `SupportedInstruments` interface, and it will have two types: either this structure or the current one.

However @icatalina is aware of this, he has all the context of this implementation.